src/game_api/rpc.cpp

```diff
@@ -448,20 +444,20 @@ void flip_entity(uint32_t uid)
 
 void set_camera_position(float cx, float cy)
 {
-    auto& state = State::get();
-    state.set_camera_position(cx, cy);
+    StateMemory* state = State::ptr();
+    state->set_camera_position(cx, cy);
 }
 
 void warp(uint8_t world, uint8_t level, uint8_t theme)

```

Comment:
All of those simple functions that just get StateMemory and return some value from it or based on it, should probably be just member functions of the StateMemory. Then in `lua_vm` just make lambda.